### ISSUE TITLE
Add support for the --recursive option

### ISSUE THREAD

**Issue by hotgazpacho** [SC5/serverless-mocha-plugin#65]
I'd like to organize my functions and tests similarly. For example:
```
|--functions
|  |--namespace
|     |--handler.js
|     |--helper.js 
|--test
|  |--namespace
|     |--handler.test.js
|     |--helper.test.js
```
The way this plugin constructs the arguments to mocha currently results in no tests being found when the structure above is being used. Mocha has a command line option, `--recursive` to allow running all tests recursively under a directory.

This plugin should recognize this as an option and pass it through to the mocha process, similar to how it does for `--compilers`

**Comment by mpuittinen** [comment]
@hotgazpacho I think that can be implemented even without a --recursive flag, i.e. would traverse the test directory by default. We'd need to think how to make sls invoke test function with the -f flag, so that the correct test is found automatically.

**Comment by mpuittinen** [comment]
@hotgazpacho This is now implemented in v 1.6.0. All subfolders of test are traversed by default (i.e. no additional options)
